src/main.c

```diff
@@ -947,19 +975,34 @@ static int open_unix_listener(h2o_configurator_command_t *cmd, yoml_t *node, str
     return -1;
 }
 
-static int open_tcp_listener(h2o_configurator_command_t *cmd, yoml_t *node, const char *hostname, const char *servname, int domain,
-                             int type, int protocol, struct sockaddr *addr, socklen_t addrlen)
+static int create_socket_bind_addr_start_listen(int domain, int type, int protocol, struct sockaddr *addr, socklen_t addrlen,
+                                                int *so_reuseport)
```

Comment:
Am I correct in assuming that we no longer need to change the definition of `open_tcp_listener` now that we create new sockets in `on_config_listen`?

Reply:
seems yes。